### ISSUE TITLE
Hatch error at death

### ISSUE THREAD

**Issue by techygrrrl** [folklorelabs/pockest-helper#116]
When a monster dies, it consistently prints out a hatch error:

```
[8/31/2024, 2:06:29 PM] Crimson Viper appears (P: 285, S: 254, T: 420)
[8/31/2024, 3:07:29 PM] Crimson Viper died 🪦
[8/31/2024, 3:07:29 PM] Crimson Viper ERROR❗Pockest Helper detected a Monster that it did not hatch. Please refrain from manually hatching monsters as this will reduce the effectiveness of Pockest Helper.
```

and

```
[9/3/2024, 8:01:17 PM] Lily appears (P: 0, S: 1114, T: 0)
[9/3/2024, 8:58:17 PM] Lily died 🪦
[9/3/2024, 8:58:17 PM] Lily ERROR❗Pockest Helper detected a Monster that it did not hatch. Please refrain from manually hatching monsters as this will reduce the effectiveness of Pockest Helper.
```

This looks more like noise than anything since the monster just died, but in case it's something else, thought I'd share.

Version: 1.2.0

**Comment by folklorelabs** [comment]
Thanks, ya this is just noise. I think I have this fixed in the newest version but haven't had a monster die/depart yet to test it.

**Comment by folklorelabs** [comment]
Should be fixed in [v1.2.2](https://github.com/folklorelabs/pockest-helper/releases/tag/v1.2.2).
